config/site.php

```diff
@@ -63,12 +63,12 @@
             'icon' => 'fa-book',
         ],
 
-        'tales' => [
-            'titleEn' => 'Tales',
-            'subtitleEn' => 'From the 250 Acre Wood',
-            'titleTh' => 'เรื่องเล่า',
-            'subtitleTh' => 'จากวัดน้อยในป่าใหญ่์',
-            'icon' => 'fa-tree',
+        'questions' => [
+            'titleEn' => 'Questions',
+            'subtitleEn' => '',
+            'titleTh' => '',
```

Comment:
Google translate shows "คำถาม" for "Question", not sure that's an appropriate translation but might be something we should ask via email to folks at the monastery.

Reply:
Agree!